rust/logging/src/lib.rs

```diff
@@ -205,10 +206,18 @@ where
     use tracing::Level;
 
     sentry_tracing::layer()
-        .event_filter(move |md| match *md.level() {
-            Level::ERROR | Level::WARN => EventFilter::Event | EventFilter::Breadcrumb | EventFilter::Log,
-            Level::INFO | Level::DEBUG => EventFilter::Breadcrumb | EventFilter::Log,
-            _ => EventFilter::Ignore,
+        .event_filter(move |md| {
+            let mut event_filter = match *md.level() {
+                Level::ERROR | Level::WARN => EventFilter::Event | EventFilter::Breadcrumb,
+                Level::INFO | Level::DEBUG => EventFilter::Breadcrumb,
+                _ => return EventFilter::Ignore,
+            };
+
+            if feature_flags::stream_logs() {
```

Comment:
Just so I'm following, the feature_flags module has logic in place to automatically update from the feature flag service?